gg-pac-team3-front/src/pages/frontPage/Panel.jsx

```diff
@@ -0,0 +1,15 @@
+import panelImg from "../../assets/images/panelImage.jpg";
+import styling from "./panel.module.css";
+
+const Panel = () => {
+    return(
```

Comment:
Now would be a good time to install a formatter like `Prettier` and turn on `format on save` and `format on paste` in your VS Code settings.

Reply:
decided to manually fix it.